samples/proxy_info.yaml

```diff
@@ -7,3 +7,4 @@ local_ota_proxy_listen_addr: 127.0.0.1
 local_ota_proxy_listen_port: 8082
 # if otaclient-logger is installed locally
 logging_server: "http://127.0.0.1:8083"
+logging_server_grpc: "http://127.0.0.1:8084"
```

Comment:
Once new otaclient with this PR is released, we will update `proxy_info.yaml` in each project(main target will be X2).